src/earn/prepareTransactions.ts

```diff
@@ -93,16 +92,22 @@ export async function prepareDepositTransactions({
   }
 }
 
-export async function prepareWithdrawAndClaimTransactions({
-  pool,
+export async function prepareWithdrawTransactionsWithSwap({
```

Comment:
@satish-ravi Tried a few ways to make this work without having this additional function but came up short.

Reply:
you can still avoid duplicating the fn by calling `prepareWithdrawTransactions` from this function and adjusting the response type right? This is exactly what we did in our pairing session except we did it with an inline function in the hook instead of an exported function here

Reply:
Was able to make this change in 8cffb5b8d2fcf4f24c47502d866e4a59c6c8a8f8 and it is a lot cleaner. I tried to do something similar with `prepareWithdrawAndClaimTransactions`, but ran into issues merging the `prepareTransactionsResult` objects.